rero/rero-ils
# data model: implement title and variants transformation

data model: implement title and variants transformation
    
    * Implements title and variants transformation for Marc21.
    * Implements title and variants transformation for Marc21 eBook.
    * Adapts ES mapping.
    * Adapts JSON schemas.
    * Adapts detailed document view.

## Why are you opening this PR?

- implement title and variants transformation

## How to test?

Go to public detail view of documents and check the display of the title.
Select arabic documents and check that Title, Variant title and Statement of responsibility are displayed if they exist in the bibliographic record.
Check other records.
records/documents/detail/34 should display all fields.

## Code review check list
